config.json

```diff
@@ -5,6 +5,16 @@
   "active": true,
   "exercises": [
     {
+       "difficulty": 1,
```

Comment:
"Hello World" is first by tradition, so I’d like to move this later in the list of exercises.